src/pathfinding_service/api.py

```diff
@@ -260,6 +260,7 @@ def get(self) -> Tuple[dict, int]:
                 "version": self.version,
                 "operator": operator,
                 "message": message,
+                "eth_address": self.pathfinding_service.address,
```

Comment:
What about `payment_address`?

Reply:
I wanted to put `eth` in there to make the distinction from an IP address. But payment address is better, since it's obviously an eth address and conveys the purpose.